src/tests/dbus-tests/test_btrfs.py

```diff
@@ -381,3 +381,20 @@ def test_subvolume_mount(self):
         dbus_mounts.assertLen(1)  # just one mountpoint
         dbus_mnt = self.ay_to_str(dbus_mounts.value[0])  # mountpoints are arrays of bytes
         self.assertEqual(dbus_mnt, mnt_path)
+
+    def test_get_default_subvolume_id(self):
+        dev = self._get_devices(1)[0]
+        self.addCleanup(self._clean_format, dev.obj)
+
+        manager = self.get_object('/Manager')
+        manager.CreateVolume([dev.obj_path],
+                             'test_subvolume_id', 'single', 'single',
+                             self.no_options,
+                             dbus_interface=self.iface_prefix + '.Manager.BTRFS')
+
+        fstype = self.get_property(dev.obj, '.Block', 'IdType')
+        fstype.assertEqual('btrfs')
+
+        default_id = dev.obj.GetDefaultSubvolumeID(self.no_options,
```

Comment:
You can use the `_temp_mount` context manager to mount the volume for this operation, the tests are now failing because the device is not mounted.